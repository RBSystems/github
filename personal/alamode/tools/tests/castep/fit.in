&general
  PREFIX = NaH
  MODE = fitting
  NAT = 54; NKD = 2
  KD = H Na
/

&fitting
  NDATA = 10001 
  NSTART = 2; NEND = 10001
  DFILE = disp.dat
  FFILE = force.dat
/

&interaction
  NORDER = 1  # 1: harmonic, 2: cubic, ..
/

&cell
  11.616360910020440
  0.0 1.0 1.0 # a1
  1.0 0.0 1.0 # a2
  1.0 1.0 0.0 # a3
/

&cutoff 
  Na-Na None None
  Na-H None None
  H-H None None
/

&position
 1      0.1666666667       0.1666666667       0.1666666667
 1      0.1666666667       0.1666666667       0.5000000000
 1      0.1666666667       0.1666666667       0.8333333333
 1      0.1666666667       0.5000000000       0.1666666667
 1      0.1666666667       0.5000000000       0.5000000000
 1      0.1666666667       0.5000000000       0.8333333333
 1      0.1666666667       0.8333333333       0.1666666667
 1      0.1666666667       0.8333333333       0.5000000000
 1      0.1666666667       0.8333333333       0.8333333333
 1      0.5000000000       0.1666666667       0.1666666667
 1      0.5000000000       0.1666666667       0.5000000000
 1      0.5000000000       0.1666666667       0.8333333333
 1      0.5000000000       0.5000000000       0.1666666667
 1      0.5000000000       0.5000000000       0.5000000000
 1      0.5000000000       0.5000000000       0.8333333333
 1      0.5000000000       0.8333333333       0.1666666667
 1      0.5000000000       0.8333333333       0.5000000000
 1      0.5000000000       0.8333333333       0.8333333333
 1      0.8333333333       0.1666666667       0.1666666667
 1      0.8333333333       0.1666666667       0.5000000000
 1      0.8333333333       0.1666666667       0.8333333333
 1      0.8333333333       0.5000000000       0.1666666667
 1      0.8333333333       0.5000000000       0.5000000000
 1      0.8333333333       0.5000000000       0.8333333333
 1      0.8333333333       0.8333333333       0.1666666667
 1      0.8333333333       0.8333333333       0.5000000000
 1      0.8333333333       0.8333333333       0.8333333333
 2     0.0000000000       0.0000000000       0.0000000000
 2    -0.0000000000       0.0000000000       0.3333333333
 2    -0.0000000000       0.0000000000       0.6666666667
 2    -0.0000000000       0.3333333333       0.0000000000
 2    -0.0000000000       0.3333333333       0.3333333333
 2     0.0000000000       0.3333333333       0.6666666667
 2    -0.0000000000       0.6666666667       0.0000000000
 2     0.0000000000       0.6666666667       0.3333333333
 2    -0.0000000000       0.6666666667       0.6666666667
 2     0.3333333333      -0.0000000000       0.0000000000
 2     0.3333333333       0.0000000000       0.3333333333
 2     0.3333333333       0.0000000000       0.6666666667
 2     0.3333333333       0.3333333333       0.0000000000
 2     0.3333333333       0.3333333333       0.3333333333
 2     0.3333333333       0.3333333333       0.6666666667
 2     0.3333333333       0.6666666667      -0.0000000000
 2     0.3333333333       0.6666666667       0.3333333333
 2     0.3333333333       0.6666666667       0.6666666667
 2     0.6666666667      -0.0000000000       0.0000000000
 2     0.6666666667       0.0000000000       0.3333333333
 2     0.6666666667       0.0000000000       0.6666666667
 2     0.6666666667       0.3333333333      -0.0000000000
 2     0.6666666667       0.3333333333       0.3333333333
 2     0.6666666667       0.3333333333       0.6666666667
 2     0.6666666667       0.6666666667       0.0000000000
 2     0.6666666667       0.6666666667       0.3333333333
 2     0.6666666667       0.6666666667       0.6666666667
