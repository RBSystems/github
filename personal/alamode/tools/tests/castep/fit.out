 +-----------------------------------------------------------------+
 +                         Program ALM                             +
 +                             Ver.  1.0.2                         +
 +-----------------------------------------------------------------+

 Number of OpenMP threads = 4

 Job started at Mon Apr 23 16:13:51 2018


 Input variables:
 -------------------------------------------------------------------
 General:
  PREFIX = NaH
  MODE = fitting
  NAT = 54; NKD = 2
  NSYM = 0; PRINTSYM = 0; TOLERANCE = 1e-06
  KD =    H  Na
  PERIODIC =   1  1  1
  MAGMOM = 
  HESSIAN = 0

 Interaction:
  NORDER = 1
  NBODY =   2

 Fitting:
  DFILE = disp.dat
  FFILE = force.dat
  NDATA = 10001; NSTART = 2; NEND = 10001; NSKIP = 0
  NBOOT = 0
  MULTDAT = 1
  ICONST = 1
  ROTAXIS = 
  FC2XML = 
  FC3XML = 

 -------------------------------------------------------------------

 SYSTEM
 ======

  Lattice Vector
    0.000000e+00   1.161636e+01   1.161636e+01 : a1
    1.161636e+01   0.000000e+00   1.161636e+01 : a2
    1.161636e+01   1.161636e+01   0.000000e+00 : a3

  Cell volume = 3.135020e+03 (a.u)^3

  Reciprocal Lattice Vector
   -2.704455e-01   2.704455e-01   2.704455e-01 : b1
    2.704455e-01  -2.704455e-01   2.704455e-01 : b2
    2.704455e-01   2.704455e-01  -2.704455e-01 : b3

  Atomic species:
     1    H
     2   Na

  Atomic positions in fractional basis and atomic species
     1   1.666667e-01   1.666667e-01   1.666667e-01    1
     2   1.666667e-01   1.666667e-01   5.000000e-01    1
     3   1.666667e-01   1.666667e-01   8.333333e-01    1
     4   1.666667e-01   5.000000e-01   1.666667e-01    1
     5   1.666667e-01   5.000000e-01   5.000000e-01    1
     6   1.666667e-01   5.000000e-01   8.333333e-01    1
     7   1.666667e-01   8.333333e-01   1.666667e-01    1
     8   1.666667e-01   8.333333e-01   5.000000e-01    1
     9   1.666667e-01   8.333333e-01   8.333333e-01    1
    10   5.000000e-01   1.666667e-01   1.666667e-01    1
    11   5.000000e-01   1.666667e-01   5.000000e-01    1
    12   5.000000e-01   1.666667e-01   8.333333e-01    1
    13   5.000000e-01   5.000000e-01   1.666667e-01    1
    14   5.000000e-01   5.000000e-01   5.000000e-01    1
    15   5.000000e-01   5.000000e-01   8.333333e-01    1
    16   5.000000e-01   8.333333e-01   1.666667e-01    1
    17   5.000000e-01   8.333333e-01   5.000000e-01    1
    18   5.000000e-01   8.333333e-01   8.333333e-01    1
    19   8.333333e-01   1.666667e-01   1.666667e-01    1
    20   8.333333e-01   1.666667e-01   5.000000e-01    1
    21   8.333333e-01   1.666667e-01   8.333333e-01    1
    22   8.333333e-01   5.000000e-01   1.666667e-01    1
    23   8.333333e-01   5.000000e-01   5.000000e-01    1
    24   8.333333e-01   5.000000e-01   8.333333e-01    1
    25   8.333333e-01   8.333333e-01   1.666667e-01    1
    26   8.333333e-01   8.333333e-01   5.000000e-01    1
    27   8.333333e-01   8.333333e-01   8.333333e-01    1
    28   0.000000e+00   0.000000e+00   0.000000e+00    2
    29  -0.000000e+00   0.000000e+00   3.333333e-01    2
    30  -0.000000e+00   0.000000e+00   6.666667e-01    2
    31  -0.000000e+00   3.333333e-01   0.000000e+00    2
    32  -0.000000e+00   3.333333e-01   3.333333e-01    2
    33   0.000000e+00   3.333333e-01   6.666667e-01    2
    34  -0.000000e+00   6.666667e-01   0.000000e+00    2
    35   0.000000e+00   6.666667e-01   3.333333e-01    2
    36  -0.000000e+00   6.666667e-01   6.666667e-01    2
    37   3.333333e-01  -0.000000e+00   0.000000e+00    2
    38   3.333333e-01   0.000000e+00   3.333333e-01    2
    39   3.333333e-01   0.000000e+00   6.666667e-01    2
    40   3.333333e-01   3.333333e-01   0.000000e+00    2
    41   3.333333e-01   3.333333e-01   3.333333e-01    2
    42   3.333333e-01   3.333333e-01   6.666667e-01    2
    43   3.333333e-01   6.666667e-01  -0.000000e+00    2
    44   3.333333e-01   6.666667e-01   3.333333e-01    2
    45   3.333333e-01   6.666667e-01   6.666667e-01    2
    46   6.666667e-01  -0.000000e+00   0.000000e+00    2
    47   6.666667e-01   0.000000e+00   3.333333e-01    2
    48   6.666667e-01   0.000000e+00   6.666667e-01    2
    49   6.666667e-01   3.333333e-01  -0.000000e+00    2
    50   6.666667e-01   3.333333e-01   3.333333e-01    2
    51   6.666667e-01   3.333333e-01   6.666667e-01    2
    52   6.666667e-01   6.666667e-01   0.000000e+00    2
    53   6.666667e-01   6.666667e-01   3.333333e-01    2
    54   6.666667e-01   6.666667e-01   6.666667e-01    2


  Time Elapsed: 0.004461 sec.

 -------------------------------------------------------------------

 SYMMETRY
 ========

  NSYM = 0 : Trying to find symmetry operations.
             Please be patient. 
             This can take a while for a large supercell.

  Number of symmetry operations = 1296
  Given system is not primitive cell.
  There are    27 translation operations.
  Primitive cell contains 2 atoms

  **Cell-Atom Correspondens Below**
  CELL |  ATOM
     1 |     1   28
     2 |     2   29
     3 |     3   30
     4 |     4   31
     5 |     5   32
     6 |     6   33
     7 |     7   34
     8 |     8   35
     9 |     9   36
    10 |    10   37
    11 |    11   38
    12 |    12   39
    13 |    13   40
    14 |    14   41
    15 |    15   42
    16 |    16   43
    17 |    17   44
    18 |    18   45
    19 |    19   46
    20 |    20   47
    21 |    21   48
    22 |    22   49
    23 |    23   50
    24 |    24   51
    25 |    25   52
    26 |    26   53
    27 |    27   54

  Time Elapsed: 0.050748 sec.

 -------------------------------------------------------------------

 INTERACTION
 ===========

  +++ Cutoff Radii Matrix in Bohr Unit (NKD x NKD matrix) +++
   HARMONIC
     None     None
     None     None


  List of neighboring atoms below.
  Format [N th-nearest shell, distance in Bohr (Number of atoms on the shell)]

    1 (  H):   1   3.87212 (  6) -  29( Na)  31( Na)  32( Na)  37( Na)
                                    38( Na)  40( Na)
               2   5.47601 ( 12) -   2(  H)   3(  H)   4(  H)   6(  H)
                                     7(  H)   8(  H)  10(  H)  12(  H)
                                    16(  H)  19(  H)  20(  H)  22(  H)
               3   6.70671 (  8) -  28( Na)  30( Na)  34( Na)  41( Na)
                                    42( Na)  44( Na)  46( Na)  50( Na)
               4   7.74424 (  6) -  15(  H)  17(  H)  18(  H)  23(  H)
                                    24(  H)  26(  H)
               5   8.65832 ( 12) -  33( Na)  35( Na)  36( Na)  39( Na)
                                    43( Na)  45( Na)  47( Na)  48( Na)
                                    49( Na)  51( Na)  52( Na)  53( Na)
               6   9.48472 (  8) -   5(  H)   9(  H)  11(  H)  13(  H)
                                    14(  H)  21(  H)  25(  H)  27(  H)
               7   11.6164 (  1) -  54( Na)

   28 ( Na):   1   3.87212 (  6) -   3(  H)   7(  H)   9(  H)  19(  H)
                                    21(  H)  25(  H)
               2   5.47601 ( 12) -  29( Na)  30( Na)  31( Na)  33( Na)
                                    34( Na)  35( Na)  37( Na)  39( Na)
                                    43( Na)  46( Na)  47( Na)  49( Na)
               3   6.70671 (  8) -   1(  H)   2(  H)   4(  H)  10(  H)
                                    18(  H)  24(  H)  26(  H)  27(  H)
               4   7.74424 (  6) -  42( Na)  44( Na)  45( Na)  50( Na)
                                    51( Na)  53( Na)
               5   8.65832 ( 12) -   5(  H)   6(  H)   8(  H)  11(  H)
                                    12(  H)  13(  H)  15(  H)  16(  H)
                                    17(  H)  20(  H)  22(  H)  23(  H)
               6   9.48472 (  8) -  32( Na)  36( Na)  38( Na)  40( Na)
                                    41( Na)  48( Na)  52( Na)  54( Na)
               7   11.6164 (  1) -  14(  H)


  List of interacting atom pairs considered for each order:

   ***HARMONIC***
    Atom     1(  H) interacts with atoms ... 
       1(  H)    2(  H)    3(  H)    4(  H)    5(  H)    6(  H)
       7(  H)    8(  H)    9(  H)   10(  H)   11(  H)   12(  H)
      13(  H)   14(  H)   15(  H)   16(  H)   17(  H)   18(  H)
      19(  H)   20(  H)   21(  H)   22(  H)   23(  H)   24(  H)
      25(  H)   26(  H)   27(  H)   28( Na)   29( Na)   30( Na)
      31( Na)   32( Na)   33( Na)   34( Na)   35( Na)   36( Na)
      37( Na)   38( Na)   39( Na)   40( Na)   41( Na)   42( Na)
      43( Na)   44( Na)   45( Na)   46( Na)   47( Na)   48( Na)
      49( Na)   50( Na)   51( Na)   52( Na)   53( Na)   54( Na)

    Number of total interaction pairs = 54

    Atom    28( Na) interacts with atoms ... 
       1(  H)    2(  H)    3(  H)    4(  H)    5(  H)    6(  H)
       7(  H)    8(  H)    9(  H)   10(  H)   11(  H)   12(  H)
      13(  H)   14(  H)   15(  H)   16(  H)   17(  H)   18(  H)
      19(  H)   20(  H)   21(  H)   22(  H)   23(  H)   24(  H)
      25(  H)   26(  H)   27(  H)   28( Na)   29( Na)   30( Na)
      31( Na)   32( Na)   33( Na)   34( Na)   35( Na)   36( Na)
      37( Na)   38( Na)   39( Na)   40( Na)   41( Na)   42( Na)
      43( Na)   44( Na)   45( Na)   46( Na)   47( Na)   48( Na)
      49( Na)   50( Na)   51( Na)   52( Na)   53( Na)   54( Na)

    Number of total interaction pairs = 54


  Time Elapsed: 0.06136 sec.

 -------------------------------------------------------------------

 FORCE CONSTANT
 ==============


  Number of  HARMONIC FCs : 24

  Time Elapsed: 0.06735 sec.

 -------------------------------------------------------------------

 CONSTRAINT
 ==========

  ICONST = 1: Constraints for translational invariance
              will be considered.

  Generating constraints for translational invariance ...
   HARMONIC ... done.
  Finished !

  Number of constraints [T-inv, R-inv (self), R-inv (cross)]:
   HARMONIC    2    0    0

  Constraints of T-inv and R-inv (self) are merged.
  If there are redundant constraints, they are removed in this process.

  Number of inequivalent constraints (self, cross) : 
   HARMONIC    2    0

  Total number of constraints = 2

  Time Elapsed: 0.069057 sec.

 -------------------------------------------------------------------

 FITTING
 =======

  Reference files
   Displacement: disp.dat
   Force       : force.dat

  NSTART = 2; NEND = 10001
  10000 entries will be used for fitting.

  MULTDAT = 1: Generate symmetrically equivalent displacement-force 
               data sets by using pure translational operations only.

  Total Number of Parameters : 24

  Calculation of matrix elements for direct fitting started ... done!

  Entering fitting routine: QRD with constraints
  QR-Decomposition has started ... finished. 

  Residual sum of squares for the solution: 19.9586
  Fitting error (%) : 99.8785

  Time Elapsed: 15.4875 sec.

 -------------------------------------------------------------------

 The following files are created:

 Force constants in a human-readable format : NaH.fcs
 Input data for the phonon code ANPHON      : NaH.xml


 Job finished at Mon Apr 23 16:14:06 2018

